Automattic/woocommerce-payments
# change type with transaction type in transaction details url

Fixes #4039 

#### Changes proposed in this Pull Request
Rename type parameter to transaction type for transaction details URL
                     








#### Testing instructions





* Go to Transactions section
* check transaction details pages . The informations should be correctly displayed. 

-------------------


**Post merge**
